node/tessel-export.js

```diff
@@ -1449,6 +1417,41 @@ function getWifiInfo() {
       });
     }
 
+    // when immediately connecting and restarting the wifi chip, it takes a few moments before an IP address is broadcast to Tessel.
+    // This function keeps checking for that IP until it's available.
+    function recursiveIP(network) {
+      setImmediate(() => {
+        childProcess.exec('ifconfig wlan0', (error, ipResults) => {
```

Comment:
You're probably correct. I think this was me not really understanding the how `childProcess.exec` works. I'll try removing it and testing tonight. :+1: